app/poros/food.rb

```diff
@@ -0,0 +1,15 @@
+class Food
+
+  attr_reader :description,
+              :brand,
+              :ingredients,
+              :gtinUpc,
+              :total
+  def initialize (foods, total)
+    @description = foods[:description]
+    @brand = foods[:brandOwner]
+    @ingredients = foods[:ingredients]
+    @gtinUpc = foods[:gtinUpc]
+    @total = total
```

Comment:
I don't mind that you are storing the total number of foods on each `Food` object, but one argument against this is that it's unnecessary to carry that data on each `Food` and it will take up space in memory. This is a very small scope app so wouldn't matter much but on a large scale app, we'd want to consider if that's the best way to handle it.